toolchain/sem_ir/stringify_type.cpp

```diff
@@ -283,6 +312,57 @@ auto StringifyTypeExpr(const SemIR::File& sem_ir, InstId outer_inst_id)
         }
         break;
       }
+      case CARBON_KIND(InterfaceWitnessAccess inst): {
+        auto witness_inst_id =
+            sem_ir.constant_values().GetConstantInstId(inst.witness_id);
+        auto witness =
+            sem_ir.insts().GetAs<FacetAccessWitness>(witness_inst_id);
+        auto witness_type_id =
+            sem_ir.insts().Get(witness.facet_value_inst_id).type_id();
+        auto facet_type = sem_ir.types().GetAs<FacetType>(witness_type_id);
+        // TODO: Support != 1 interface better.
+        if (auto impls_constraint = sem_ir.facet_types()
+                                        .Get(facet_type.facet_type_id)
+                                        .TryAsSingleInterface()) {
+          step_stack.PushString(")");
+          auto interface =
```

Comment:
```suggestion
          const auto& interface =
```
Avoid copying the `Interface` object.